torch_xla/csrc/runtime/pjrt_registry.cc

```diff
@@ -87,7 +87,7 @@ InitializePjRt(const std::string& device_type) {
              device_type == "ROCM") {
     TF_VLOG(1) << "Initializing PjRt GPU client...";
     bool async = sys_util::GetEnvBool(env::kEnvPjrtAsyncGpuClient, true);
-    int local_process_rank = sys_util::GetEnvInt(env::kEnvPjRtLocalRank, 0);
+    int local_process_rank = sys_util::GetEnvInt("LOCAL_RANK", 0);
```

Comment:
`LOCAL_RANK` isn't always set in `xmp.spawn` due to compatiblity issues. What about `sys_util::GetEnvInt(env::kEnvPjRtLocalRank, sys_util::GetEnvInt(env::"LOCAL_RANK", 0););`? The same thing should work for `WORLD_SIZE` with `PjRtLocalWorldSize`.

Reply:
Okay. By the way, is it possible to align all distributed environment variables with torchrun, so that we don't need to deal with such issues?

Reply:
This is what we did originally. I changed it because it caused some issue in HuggingFace's Trainer, which was relying on the presence of `LOCAL_RANK` for some unrelated behavior: https://github.com/pytorch/xla/pull/4646

Not sure if this issue is still relevant. I would prefer to use `LOCAL_RANK` and `LOCAL_WORLD_SIZE` in principle though.